src/js/ColumnUI.js

```diff
@@ -30,6 +34,18 @@ class ColumnUI {
       new CardUI(card).renderCard(cardList);
     });
   }
+
+  dnd(cardList) {
+    const getStatus = () => this.status;
+    return new Sortable(cardList, {
+      group: 'shared',
+      animation: 150,
+      onAdd({ item }) {
+        const id = item.dataset.cardId;
+        CardAPI.updateCardStatus(id, getStatus());
```

Comment:
```suggestion
        CardAPI.updateCard({
          id,
          status: getStatus()
        });
```

maybe better to use existing func `updateCard`

Reply:
> maybe better to use existing func `updateCard`

yep, sure

Reply:
fixed in f55bcf9